### ISSUE TITLE
[💡SUG] Customize Dataset Splitting

### ISSUE THREAD

**Issue by Xueqi-Li** [RUCAIBox/RecBole#1945]
I observed that Recbole supports two dataset splitting strategies: Leave-one-out and Ratio-based.

My intention is to designate interactions occurring before a specified time as the training set, while considering the rest as the test set. I am curious to know if there is any API that facilitates this splitting strategy or any possible solutions? Thank you.

**Comment by TayTroye** [comment]
@Xueqi-Li 
Hello,you can write one function like  `split_by_ratio`  to accomplish this splitting strategy .https://github.com/RUCAIBox/RecBole/blob/4121d5c2a7c4813566d2240e5d474d6ce2f0929f/recbole/data/dataset/dataset.py#L1623
